sled-agent/src/rack_setup/service.rs

```diff
@@ -2,23 +2,84 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-//! Rack Setup Service implementation
-
-use super::config::{HardcodedSledRequest, SetupServiceConfig as Config};
-use crate::bootstrap::config::BOOTSTRAP_AGENT_PORT;
+//! Rack Setup Service (RSS) implementation
+//!
+//! RSS triggers the initialization of:
+//! - Sled Agents (giving them underlay addresses)
+//! - Trust Quorum (coordinating between Sled Agents)
+//! - Services (such as internal DNS, CRDB, Nexus)
+//! - DNS records for those services
+//! - Handoff to Nexus, for control of Control Plane management
+//!
+//! # Phases and Configuration Files
+//!
+//! Rack setup occurs in distinct phases which are denoted by the prescence of
+//! configuration files.
+//!
+//! - /var/oxide/rss-sled-plan.toml (Sled Plan)
+//! - /var/oxide/rss-service-plan.toml (Service Plan)
+//! - /var/oxide/rss-plan-completed.marker (Plan Execution Complete)
+//!
+//! ## Sled Plan
+//!
+//! When RSS starts, it presumably is executing on a single sled, and must
```

Comment:
I'm not sure what "it presumably is executing on a single sled" means or is getting at. There are instances of RSS running on both scrimlets, right? But only one of them will actually perform setup (whichever one the technician happens to connect to / plug in to).

Reply:
Updated comment in 362f7e13.

I agree with your comment here - we expect the operator to only be initializing the sled via a connection through a single technician port.